mozregression/main.py

```diff
@@ -197,7 +198,9 @@ def _print_resume_info(self, handler):
         for i in reversed(indexes_to_remove):
             del argv[i]
 
-        if isinstance(handler, NightlyHandler):
+        argv.append('--repo=%s' % handler.build_range[0].repo_name)
+
+        if hasattr(handler, 'good_date'):
```

Comment:
I changed isinstance to hasattr for testing purposes (hard to deal with mocked instance types), but this is equivalent.